sdk/storage/azure-storage-blob/tests/test_block_blob_async.py

```diff
@@ -184,7 +213,11 @@ async def test_upload_from_file_to_blob_with_oauth(self, **kwargs):
             # Assert
             assert destination_blob_data == source_data
         finally:
-            await share_service_client.delete_share(share_client.share_name)
+            async with aiohttp.ClientSession() as requests:
```

Comment:
Does `aiohttp` always need a session? If not, we could probably do delete without one.